integration/setup/client.go

```diff
@@ -33,26 +33,43 @@ import (
 
 // mongoDBURIOpts represents mongoDBURI's options.
 type mongoDBURIOpts struct {
+	baseURI        string
```

Comment:
I will like we are going back and forth there.

`mongoDBURI` returns a MongoDB URI for the given options. Those options are created only by the in-process listener, where no URI exists and we need to create one.

When URI exists (for example, when the target system is not in-process FerretDB), we just use it with extra query options for specific tests if they need them.

With this change, `mongoDBURI` could be used to construct MongoDB URI from many different options, making it much harder to understand.

Reply:
Okie, so mongoDBURI should not be modified because it's for in-process listener and parameters should stay vanilla. Once mongoDBURI is built, then I can use that uri to add extra query options then.

Reply:
We could probably rename that function to make it more clear

Reply:
Any preference on the name?  `InProcessFerretDBURI`?

Reply:
I don't know :)